packages/@aws-cdk/aws-ssm/lib/parameter.ts

```diff
@@ -238,7 +238,7 @@ export class StringParameter extends ParameterBase implements IStringParameter {
       provider: cxapi.SSM_PARAMETER_PROVIDER,
       props: { parameterName },
       dummyValue: `dummy-value-for-${parameterName}`
-    });
+    }).value;
```

Comment:
This makes it read `getValue(/*...*/).value`... 🙁

So like... I'm getting the value from the value?

Reply:
The struct `GetContextValueResult` was already defined but unused. I had assumed it would be intended to be used here, but forgotten to be implemented (similar to how `getKey()` returns a `GetContextKeyResult`).

Also, opens up a nice avenue for returning additional metadata associated with the value in the future.

Not really any different from how any API call titled `GetFoo` returns a struct that looks like `{ foo: Foo }`.